functions-and-scope.js

```diff
@@ -14,6 +14,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+
+const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+    let counter = null;
+    for (let i = 0; i < grades.length; i++) {
+        if (grades[i] > 7) {
+            counter += 1;
```

Comment:
Dit zou je nog iets kunnen versimpelen
```suggestion
            counter++;
```